public/app/custom_playlist/customPlaylist.js

```diff
@@ -124,9 +131,21 @@ angular.module('turntify.player')
   $scope.$on('playlistCollectionUpdated', function(event){
     console.log("playlistUpdated received!");
     vm.customPlaylist = PlayerService.customPlaylist.slice(0,9);
+    vm.fullCustomPlaylistLength = PlayerService.customPlaylist.length;
+    if (vm.fullCustomPlaylistLength > 1) {
+      vm.songsOrSong = 'songs';
+    } else {
+      vm.songsOrSong = 'song';
+    }
   });
   $scope.$on('loadAllMatches', function(event){
     console.log("loadingAllMatches!");
     vm.customPlaylist = PlayerService.customPlaylist;
+    vm.fullCustomPlaylistLength = PlayerService.customPlaylist.length;
+    if (vm.fullCustomPlaylistLength > 1) {
```

Comment:
you're doing the same thing 3 times in this file -- abstract it out into a helper function